homeassistant/components/media_player/yamaha.py

```diff
@@ -46,10 +46,16 @@
     vol.Optional(CONF_SOURCE_NAMES, default={}): {cv.string: cv.string},
 })
 
+KNOWN_RECEIVERS = set()
 
+
+# pylint doesn't understand the above set() as an assignment...
+#
+# pylint: disable=global-variable-not-assigned
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the Yamaha platform."""
     import rxv
+    global KNOWN_RECEIVERS
```

Comment:
Fixed in latest change set, let me know if there is a different way you want me to handle namespacing.